src/endpoints/Cart.ts

```diff
@@ -3,93 +3,46 @@ import { AddItem, CartClass, CouponCode, SetQuantity } from '../interfaces/endpo
 import { IOrder } from '../interfaces/Order'
 import { IToken } from '../interfaces/Token'
 import { Routes } from '../routes'
+import { GET, POST, PATCH, DELETE } from '../constants'
 
 export default class Cart extends Http implements CartClass {
   public async show(token: IToken): Promise<IOrder> {
     this.spreeTokens = token
-
-    try {
-      const res = await this.get(Routes.cartPath(), {})
-      return await res.data
-    } catch (err) {
-      console.error(err)
-    }
+    return await this.spreeResponse(GET, Routes.cartPath())
   }
 
   public async create(token?: IToken): Promise<IToken> {
```

Comment:
Wrong generic type. `create` doesn't return `Promise<IToken>`. It returns a JSON:API response.